profiles/common/modules/features/multisite_mediagallery_standard/multisite_mediagallery_standard.info

```diff
@@ -3,7 +3,4 @@ description = "Contributors can add video and pictures in galleries."
 core = 7.x
 package = "Multisite Features"
 dependencies[] = "multisite_mediagallery_core"
-features[features_api][] = api:2
-features[ctools][] = context:context:3
-features[context][] = gallery_posts
```

Comment:
We are suppose to have a media_gallery context, it's exported here
https://github.com/ec-europa/platform-dev/blob/master/profiles/common/modules/features/multisite_mediagallery_core/multisite_mediagallery_core.context.inc
We should not just remove the lines, but fix the problem with the context export.

Reply:
This is indeed an issue because the context file lives in mediagallery_core but then is defined in mediagallery_standard. I didn't even realize this was the case so thats why I removed it. You are right this should be fixed but then again, I do believe is out of the scope.

Reply:
I will create a follow up ticket for this one!